development/mimir-ingest-storage/docker-compose.jsonnet

```diff
@@ -35,28 +38,25 @@ std.manifestYamlDoc({
       extraArguments: ['-ingester.ring.instance-availability-zone=zone-a'],
       extraVolumes: ['.data-mimir-write-zone-a-3:/data:delegated'],
     }),
-
-    // Zone-b.
-    'mimir-write-zone-b-1': mimirService({
-      name: 'mimir-write-zone-b-1',
-      target: 'write',
-      publishedHttpPort: 8011,
-      extraArguments: ['-ingester.ring.instance-availability-zone=zone-b'],
-      extraVolumes: ['.data-mimir-write-zone-b-1:/data:delegated'],
-    }),
-    'mimir-write-zone-b-2': mimirService({
-      name: 'mimir-write-zone-b-2',
-      target: 'write',
-      publishedHttpPort: 8012,
-      extraArguments: ['-ingester.ring.instance-availability-zone=zone-b'],
-      extraVolumes: ['.data-mimir-write-zone-b-2:/data:delegated'],
-    }),
-    'mimir-write-zone-b-3': mimirService({
-      name: 'mimir-write-zone-b-3',
-      target: 'write',
-      publishedHttpPort: 8013,
-      extraArguments: ['-ingester.ring.instance-availability-zone=zone-b'],
-      extraVolumes: ['.data-mimir-write-zone-b-3:/data:delegated'],
+    'mimir-write-zone-c-61': mimirService({
+      name: 'mimir-write-zone-c-61',
+      target: 'ingester',
+      publishedHttpPort: 8064,
+      extraArguments: [
+        '-ingester.ring.instance-availability-zone=zone-c',
+        '-ingester.ring.instance-id=ingester-zone-c-61',
+        '-ingester.partition-ring.prefix=exclusive-prefix',
+        '-ingester.ring.prefix=exclusive-prefix',
```

Comment:
(nit) I'd add a one-line comment about this ring prefixes. Otherwise, I doubt anyone joining ingest-storage development will have any context of what this `zone-c-61` instance is doing here.